From 7e4b36e3181ecbaa0e7d629b904c1a8fa63c80dd Mon Sep 17 00:00:00 2001
From: Ben Peled <bpeled@marvell.com>
Date: Thu, 18 Jul 2019 15:17:28 +0300
Subject: [PATCH 1191/1200] marvell: cn913x: env: don't force cpuidle off.
 Cpuidle off was forced since it was unsupported on CN913X in the past. This
 patch enables cpuidle for CN913x SoC family which is supported.

Change-Id: I60a3ae00998fb985315b11d4395088d7fc319a0e
Signed-off-by: Ben Peled <bpeled@marvell.com>
Reviewed-on: https://sj1git1.cavium.com/12715
Tested-by: sa_ip-sw-jenkins <sa_ip-sw-jenkins@marvell.com>
Reviewed-by: Kostya Porotchkin <kostap@marvell.com>
---
 arch/arm/mach-mvebu/armada8k/soc.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm/mach-mvebu/armada8k/soc.c b/arch/arm/mach-mvebu/armada8k/soc.c
index 8395e52530..322668c816 100644
--- a/arch/arm/mach-mvebu/armada8k/soc.c
+++ b/arch/arm/mach-mvebu/armada8k/soc.c
@@ -306,7 +306,7 @@ int arch_misc_init(void)
 	 * The variable 'type', which is returned by
 	 * get_soc_type_rev() holds these bits.
 	 */
-	if (type == A8040_DEVICE_ID || type == CN9130_DEVICE_ID)
+	if (type == A8040_DEVICE_ID)
 		env_set("cpuidle", "cpuidle.off=1");
 
 	return 0;
-- 
2.22.0

